src/sgame/sg_bot_parse.cpp

```diff
@@ -339,7 +334,7 @@ static const struct AIConditionMap_s
 } conditionFuncs[] =
 {
 	{ "alertedToEnemy",    VALUE_INT,   alertedToEnemy,    0 },
-	{ "alienMomentum",     VALUE_INT,   alienMomentum,     0 },
+	{ "momentum",          VALUE_INT,   momentum,          0 },
```

Comment:
What if you wanted to use the enemy team's momentum in the behavior tree? I guess that's the point of having both.

Reply:
Good point, but in that case, it'd be better to add an argument I think. Currently, none of those are used, though, and I fail to see how they could be used considering current capabilities of the BT mechanisms (can't do math operations notably).

Reply:
I imagine the use case for momentum would be to check whether some weapon/class/structure is unlocked. You wouldn't need any math for that - it's just comparing greater or less than a constant number. And a team-generic one wouldn't be particularly useful since the different teams have different unlocks.

Reply:
Right, but it would allow to move the non-generic part into behavior trees, which have to be specific anyway (since different gameplay for team is the goal). The C++ code could benefit from a `momentum( some_team )` function, so that there is less code there. Plus, I don't think currently a team can know opponent's momentum for players, so why bots should be allowed to do so?